docs/3x/PyTorch.md

```diff
@@ -0,0 +1,230 @@
+
+For more details, please refer to [link](https://github.com/intel/neural-compressor/discussions/1527) in Neural Compressor discussion space.
+
+So far, `neural_compressor.torch` still relies on the backend to generate the quantized model and run it on the corresponding backend, but in the future, neural_compressor is committed to provide generalized device-agnostic Q-DQ model, so as to realize one-time quantization and arbitrary deployment.
+
+## Torch-like APIs
+
+Currently, we provide below three user scenarios, through `prepare`&`convert`, `autotune` and `load` APIs.
+
+- One-time quantization of the model
+- Get the best quantized model by setting the search scope and target
+- Direct deployment of the quantized model
+
+### Quantization APIs
+
+```python
+def prepare(
+    model: torch.nn.Module,
+    quant_config: BaseConfig,
+    inplace: bool = True,
+    example_inputs: Any = None,
+):
+    """Prepare the model for calibration.
+
+    Insert observers into the model so that it can monitor the input and output tensors during calibration.
+
+    Args:
+        model (torch.nn.Module): origin model
+        quant_config (BaseConfig): path to quantization config
+        inplace (bool, optional): It will change the given model in-place if True.
+        example_inputs (tensor/tuple/dict, optional): used to trace torch model.
+
+    Returns:
+        prepared and calibrated module.
+    """
+```
+
+```python
+def convert(
+    model: torch.nn.Module,
+    quant_config: BaseConfig = None,
+    inplace: bool = True,
+):
+    """Convert the prepared model to a quantized model.
+
+    Args:
+        model (torch.nn.Module): the prepared model
+        quant_config (BaseConfig, optional): path to quantization config, for special usage.
+        inplace (bool, optional): It will change the given model in-place if True.
+
+    Returns:
+        The quantized model.
+    """
+```
+
+### Autotune API
+
+```python
+def autotune(
+    model: torch.nn.Module,
+    tune_config: TuningConfig,
+    eval_fn: Callable,
+    eval_args=None,
+    run_fn=None,
+    run_args=None,
+    example_inputs=None,
+):
+    """The main entry of auto-tune.
+
+    Args:
+        model (torch.nn.Module): _description_
+        tune_config (TuningConfig): _description_
+        eval_fn (Callable): for evaluation of quantized models.
+        eval_args (tuple, optional): arguments used by eval_fn. Defaults to None.
+        run_fn (Callable, optional): for calibration to quantize model. Defaults to None.
+        run_args (tuple, optional): arguments used by run_fn. Defaults to None.
+        example_inputs (tensor/tuple/dict, optional): used to trace torch model. Defaults to None.
+
+    Returns:
+        The quantized model.
+    """
+```
+
+### Load API
```

Comment:
do we consider exposing only load in user-facing API?  save functionality is not planned to expose?

Reply:
The save API is linked to model, using `model.save()`. we don't have unified API currently.
I will write a brief intro here to let user know that.